app/src/main/java/com/example/marvelapp/adapter/CharacterAdapter.kt

```diff
@@ -7,14 +7,13 @@ import androidx.recyclerview.widget.RecyclerView
 import com.bumptech.glide.Glide
 import com.example.marvelapp.R
 import com.example.marvelapp.databinding.CardviewCharacterBinding
-import com.example.marvelapp.entity.Character
 
 interface CharacterListener {
     fun setOnClickListener(characterId: String)
 }
 
 class CharacterAdapter(
-    private val characters: List<Character>,
+    private val characters: List<com.example.domain.entity.Character>,
```

Comment:
add the correct import instead of writing the path here

Reply:
done

Reply:
done